### DISCUSSION TITLE
Animation demo

### DISCUSSION THREAD

**Discussion by knpwrs** [dflex-js/dflex#581] [Q&A]
The homepage says that "animated transformation with each interaction" can be done with dflex; however, the demo doesn't have any animation. Can animations be added to the demo or to a separate demo?

**Comment by jalal246** [comment]
Hi @knpwrs. The internal drag and drop is done with CSS transform and inside an animated frame. If you have [commit ](https://github.com/dflex-js/dflex#commit-default-value) to false then DFlex will keep it this way forever.  However, Custom CSS animation is not provided as a built-in feature you can simply add  `transition: transform <duration> <timing-function> <delay>`  to the elements that will be transformed and it will work as required. I used  `transition: transform 0.2s cubic-bezier(0.2, 0, 0, 1) 0s` for the playground. But DFlex won't enforce a specific duration or timing function. What it does, is provide the mechanism to let you do it.
